src/ThreeEditor/js/Viewport.js

```diff
@@ -29,6 +29,18 @@ export function Viewport(
     }
 
     let sceneViewHelpers = new THREE.Scene();
+    let clippedScene = new THREE.Scene();
+
+    var skyColor = 0x00aaff; // Deep Sky Blue
+    var groundColor = 0xffaa00; // Orange
+    var intensity = 1;
+
+    var light = new THREE.HemisphereLight(skyColor, groundColor, intensity);
+    light.name = 'HemisphereLight';
+
+    light.position.set(10000, 10000, 0);
```

Comment:
Assuming units of [cm] the light is locates at more than 100 meters from the origin of coordinate system, at [100,100,0] m. Why is it so far away ?